modules/images/webp-uploads/helper.php

```diff
@@ -20,6 +20,14 @@ function webp_uploads_get_upload_image_mime_transforms() {
 		'image/webp' => array( 'image/webp' ),
 	);
 
+	// Check Setting for generating mimes.
+	if ( true === (bool) get_option( 'generate_webp_and_jpeg' ) ) {
```

Comment:
We should always prefix options based on where they're coming from (core, which plugin, which theme). Let's prefix the setting/option name with `perflab_`.